content/pt-br/docs/concepts/policy/node-resource-managers.md

```diff
@@ -0,0 +1,22 @@
+---
+reviewers:
+- derekwaynecarr
+- klueska
```

Comment:
The reviewers section should be removed in the translated version.

Reply:
Ok, fix!